ccw.core.test/src/java/ccw/core/BotUtils.java

```diff
@@ -98,7 +99,11 @@ public BotUtils fillNewProject(SWTBot bot, String projectName) throws Exception
 			location.setText(testLocation);
 		}
 		bot.button("Finish").click();
-		waitForWorkspace();
```

Comment:
Mmm according to my researches it is not necessari, the test was faling because it cannot open a menu ( not in foreground)...this is just temporary, then in the other PR i really enhanced BotUtils